test/common/stats/real_thread_test_base.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "source/common/stats/thread_local_store.h"
+#include "source/common/thread_local/thread_local_impl.h"
+
+#include "test/mocks/thread_local/mocks.h"
+#include "test/test_common/real_threads_test_helper.h"
+
+namespace Envoy {
+namespace Stats {
+
+class ThreadLocalStoreNoMocksTestBase {
+public:
+  ThreadLocalStoreNoMocksTestBase();
+  ~ThreadLocalStoreNoMocksTestBase();
+
+  StatName makeStatName(absl::string_view name);
+
+  SymbolTableImpl symbol_table_;
+  AllocatorImpl alloc_;
+  ThreadLocalStoreImplPtr store_;
+  Scope& scope_;
+  StatNamePool pool_;
+};
+
+// Helper base class for thread local stats testing in multiple-workers setup.
+// This is used in thread local store tests.
+class ThreadLocalRealThreadsTestBase : public Thread::RealThreadsTestHelper,
```

Comment:
again, this is no longer a TestBase, Rename to *TestMixin

Reply:
done